### DISCUSSION TITLE
Clarification on the buyer_coupon Field in the Transactions

### DISCUSSION THREAD

**Discussion by joselufo** [etsy/open-api#1143] [Q&A]
I am currently working with the Etsy V3 API, specifically the Transactions object, and I came across the field named buyer_coupon. However, I do not understand what's the attribution of this value because I can't relate it to any number in my orders on Etsy and the official documentation does not say much. 

Could someone provide clarification on what the buyer_coupon field represents within the Transactions object? What kind of data does it hold, and in what scenarios would this field be populated? Is there a particular format for the data contained in this field? Any insights or references to relevant documentation would be greatly appreciated.

**Comment by Vincnet26** [comment]
Hi,
all I can tell you is that when there is a promo code, this field is filled with the discount amount per item. The same amount is in "shop_coupon"

**Reply by joselufo** [reply]
I have examples in one of the shops where the shop_coupon and the buyer_coupon have different values, that's why I don't understand the difference and the API Reference does not clarify it.

**Reply by Vincnet26** [reply]
So, you should be able to see what these 2 amounts correspond to by comparing them to those present on the Etsy store manager ?

**Reply by joselufo** [reply]
The problem is that if I go to the order in the Etsy store manager, I only see the discount of the `shop_coupon` and matches in the Etsy UI but I do not see anything related to the `buyer_coupon`.

**Reply by Gareth064** [reply]
Maybe the `shop_coupon` is when the buyer users a coupon setup by the shop/seller, and the `buyer_coupon` is when Etsy do an offer, giving all buyers money off when they use the coupon setup by Etsy.

**Reply by Vincnet26** [reply]
I just saw 2 differents amounts. It seems it's only in the case the delivery country is not in europe. I think the buyer_coupon is in the customer currency !

**Reply by joselufo** [reply]
Ok, I'll try to find an order with different quantities and research if potentially is the customer currency.
